src/client/aws_translation_client.py

```diff
@@ -6,7 +6,9 @@
 class AwsTranslateClient(object):
 
     def __init__(self):
-        load_dotenv()
+        # Load .env file if it exists (for local development)
+        if os.path.exists('.env'):
+            load_dotenv()
```

Comment:
This should not live here. This should be in main.

Reply:
TBR with factory method